chromium_src/chrome/browser/policy/configuration_policy_handler_list_factory.cc

```diff
@@ -69,7 +18,7 @@ std::unique_ptr<ConfigurationPolicyHandlerList> BuildHandlerList(
   std::unique_ptr<ConfigurationPolicyHandlerList> handlers =
       BuildHandlerList_ChromiumImpl(chrome_schema);
 
-  for (const auto& entry : kBraveSimplePolicyMap) {
+  for (const auto& entry : policy::kBraveSimplePolicyMap) {
```

Comment:
`policy::` not necessary. this function is already in `policy` namespace.

Reply:
done